src/test/java/com/mekomsolutions/eip/route/OdooPatientHandlerSyncsAnyCreatedPatientEventRouteTest.java

```diff
@@ -0,0 +1,130 @@
+package com.mekomsolutions.eip.route;
+
+import static com.mekomsolutions.eip.route.OdooTestConstants.EX_PROP_ODOO_OP;
+import static com.mekomsolutions.eip.route.OdooTestConstants.EX_PROP_ODOO_PATIENT_ID;
+import static com.mekomsolutions.eip.route.OdooTestConstants.ODOO_OP_CREATE;
+import static com.mekomsolutions.eip.route.OdooTestConstants.PATIENT_UUID;
+import static com.mekomsolutions.eip.route.OdooTestConstants.URI_PATIENT_HANDLER;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.openmrs.eip.mysql.watcher.WatcherConstants.PROP_EVENT;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.camel.EndpointInject;
+import org.apache.camel.Exchange;
+import org.apache.camel.builder.AdviceWithRouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.support.DefaultExchange;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.eip.AppContext;
+import org.openmrs.eip.mysql.watcher.Event;
+import org.springframework.test.context.TestPropertySource;
+
+import ch.qos.logback.classic.Level;
+
+@TestPropertySource(properties = {"openmrs.identifier.type.uuid=" + OdooPatientHandlerSyncsAnyCreatedPatientEventRouteTest.ID_TYPE_UUID, "should.sync.any.patient.asCustomer=true"})
+public class OdooPatientHandlerSyncsAnyCreatedPatientEventRouteTest extends BaseOdooRouteTest {
```

Comment:
The test belongs in OdooPatientHandlerRouteTest class, please move it and delete this class

Reply:
This was the easier way to have a test with an application property set differently. Also discussed [here](https://stackoverflow.com/a/48570867).

Reply:
You are just running away from the issue instead of addressing it, the way you deal with it is by setting the property programmatically at the test method level see spring's [TestPropertySourceUtils.addInlinedPropertiesToEnvironment](url)

Reply:
Ok, thanks @wluyima. This still fails for as long as I'm accessing the property as an extrapolation like this[`{{create.customer.if.not.exist}}`]. I'll change to access the property using the simple spring expression language instead i.e like this[`${properties:create.customer.if.not.exist:null}`] to have this work in tests.